test/index.ts

```diff
@@ -5,18 +5,19 @@
 // By default the test runner in use is Mocha based.
 //
 // You can provide your own test runner if you want to override it by exporting
-// a function run(testRoot: string, clb: (error:Error) => void) that the extension
-// host can call to run the tests. The test runner is expected to use console.log
-// to report the results back to the caller. When the tests are finished, return
-// a possible error to the callback or null if none.
+// a function run(testRoot: string, clb: (error:Error) => void) that the
```

Comment:
if we can avoid modifying this file, that would be better.

Add it to ignore list if you need to.